panel/widgets/chat.py

```diff
@@ -203,9 +203,10 @@ class ChatEntry(CompositeWidget):
     show_timestamp : bool
         Whether to display the timestamp of the message.
     """
+    _ignored_refs: ClassVar[Tuple[str,...]] = ('value',)
```

Comment:
This solved the issue with `Button`s and other widgets being turned into `str`s @ahuang11 and @philippjfr .

I believe its the right solution. But I'm not familiar with this `attribute` and why its there?